aezeed/cipherseed_rpctest.go

```diff
@@ -0,0 +1,13 @@
+// +build rpctest
```

Comment:
Any reason to gate the unit tests with this flag as well? It'd be nice to have them when just running `make unit` as well.

Reply:
Yes, the unit test already has the fast options. The thing is, for the itest, the `lnd-itest` binary is built with `go build` and not `go test`. So this code needs to be in a non-test file but guarded with the build flag to only be picked up in itests.
But I added the options to the unit tests in the macaroons package, that shaves another 300ms off of the tests there.